### ISSUE TITLE
[1.14.4] Config resets to defaults

### ISSUE THREAD

**Issue by Wynadorn** [squeek502/AppleSkin#50]
Hey Squeek502, something might have caused an issue regarding the config file in Forge.
The config seems to be resetting to defaults.

```
21:36:45 [modloading-worker-4/WARN] [ne.mi.co.ForgeConfigSpec/CORE] Configuration file config\appleskin.toml is not correct. Correcting
21:36:45 [modloading-worker-4/WARN] [ne.mi.co.ForgeConfigSpec/CORE] Incorrect key client was corrected from null to SimpleCommentedConfig:{}
```

Log from a new server:
https://pastebin.com/raw/fBvZ5C1d

minecraft_server.1.14.4.jar
forge-1.14.4-28.1.1.jar
AppleSkin-mc1.14.4-forge-1.0.12.jar

contents of appleskin.toml:
https://pastebin.com/raw/sYfUAC9B
```

[client]
	#If true, shows the hunger (and saturation if showSaturationHudOverlay is true) that would be restored by food you are currently holding
	showFoodValuesHudOverlay = true
	#If true, shows your food exhaustion as a progress bar behind the hunger bars
	showFoodExhaustionHudUnderlay = true
	#If true, adds a line that shows your hunger, saturation, and exhaustion level in the F3 debug overlay
	showFoodStatsInDebugOverlay = true
	#If true, shows the hunger and saturation values of food in its tooltip while holding SHIFT
	showFoodValuesInTooltip = true
	#If true, shows the hunger and saturation values of food in its tooltip automatically (without needing to hold SHIFT)
	showFoodValuesInTooltipAlways = true
	#If true, shows your current saturation level overlayed on the hunger bar
	showSaturationHudOverlay = true
```

Also reported by jiooocen but it the comment section.
![image](https://user-images.githubusercontent.com/4095127/64926750-80dc5100-d801-11e9-8a33-1c27982567e5.png)

**Comment by gravysamich** [comment]
this issue still exists in the most recent version of the mod

**Comment by squeek502** [comment]
Should be fixed in 1.0.13: https://www.curseforge.com/minecraft/mc-mods/appleskin/files/2832298
